frontend/public/components/nav.jsx

```diff
@@ -49,7 +49,7 @@ class NavLink extends React.PureComponent {
 class ResourceNSLink extends NavLink {
   static isActive (props, resourcePath, activeNamespace) {
     const href = stripNS(formatNamespacedRouteForResource(props.resource, activeNamespace));
-    return resourcePath === href || _.startsWith(resourcePath, `${href}/`) || (props.model && resourcePath === referenceForModel(props.model));
+    return resourcePath === href || _.startsWith(resourcePath, `${href}/`) || (props.model && _.startsWith(resourcePath, referenceForModel(props.model)));
```

Comment:
Does `referenceForModel` include the trailing slash? Wondering if we might have a false positive if more than one resource has the same prefix (for instance, `/build` and `/buildconfig`)

Reply:
Fixed